wolfssl/wolfcrypt/types.h

```diff
@@ -31,9 +31,6 @@ decouple library dependencies with standard string, memory and so on.
 #ifndef WOLF_CRYPT_TYPES_H
 #define WOLF_CRYPT_TYPES_H
 
-    #ifdef HAVE_CONFIG_H
```

Comment:
Can you double check that all .c files in the library proper have this?

Reply:
Looked like they all did. I did however find more references to config.h that I removed/replaced with the appropriate header